frontend/pages/DashboardPage/cards/ActivityFeed/GlobalActivityItem/GlobalActivityItem.tsx

```diff
@@ -17,22 +17,6 @@ import { ShowActivityDetailsHandler } from "components/ActivityItem/ActivityItem
 
 const baseClass = "global-activity-item";
 
-const PREMIUM_ACTIVITIES = new Set([
```

Comment:
No longer used?

Reply:
nope this was unused. premium copy is handled in the activity template function now.